corehq/tests/pytest_plugins/redislocks.py

```diff
@@ -48,6 +49,8 @@ class TestLock:
     lock = attr.ib(repr=False)
     timeout = attr.ib()
 
+    local = Mock()
```

Comment:
This change is to allow setting `lock.local.token = lock_token`.

Reply:
Might be useful to add a comment so future readers of this code understand why it's there. Can be done in a separate PR after this one is merged.

Reply:
6e803b154b9